frontend/src/router/index.ts

```diff
@@ -51,7 +51,8 @@ const routes = [
         component: LandingView,
     },
     {
-        path: routeItems.dashboard.path,
+        // a param can be made optional by adding "?"
+        path: `${routeItems.dashboard.path}/:newUserAccessIds?/:newAppAdminId?/:newDelegatedAdminIds?`,
```

Comment:
I do not think this is the intended usage from vue-router, although the "?" means optional.
This looks like a path matching but you have used it as optional and combined with several of them, it has no meaning if it does work (such as `/dashboard/` or `/dashboard/1/`)
I think this still needs to be changed.

Reply:
Hi Ian, please check the router again, but basically updated to use the query parameter. The only thing is that when I try to access to `/dashboard/`, it doesn't work. But `/dashboard?newUserAccessIds=36,37` works fine.  I can show you tomorrow

Reply:
checkout to the main branch, when run it local got the same error, but FAM-DEV which runs on main branch works fine. So will ignore the local case.